src/cephadm/cephadm

```diff
@@ -4884,10 +4884,12 @@ def command_shell(ctx):
             mount = pathify(split_src_dst[0])
             filename = os.path.basename(split_src_dst[0])
             if len(split_src_dst) > 1:
-                dst = split_src_dst[1] + ':z' if len(split_src_dst) == 3 else split_src_dst[1]
+                dst = split_src_dst[1]
+                if len(split_src_dst) == 3:
+                    dst = '{}:{}'.format(dst, split_src_dst[2])
                 mounts[mount] = dst
             else:
-                mounts[mount] = '/mnt/{}:z'.format(filename)
+                mounts[mount] = '/mnt/{}'.format(filename)
```

Comment:
are we sure this won't cause an issue with SELinux?

similar to: https://tracker.ceph.com/issues/44701

Reply:
this doesn't prevent from adding `:z` when it's needed

Reply:
ah yep and can also set to `:ro` etc which is nice!